_posts/2021-04-26-4.0.0-release.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "Apache Arrow 4.0.0 Release"
+date: "2021-04-26 00:00:00 -0600"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 4.0.0 release. This covers
+3 months of development work and includes [**711 resolved issues**][1]
+from [**114 distinct contributors**][2]. See the Install Page to learn how to
+get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 3.0.0 release, Yibo Cai has been invited as a committer to Arrow, and Andrew Lamb and Jorge Leitão have joined the Project Management Committee (PMC). Thank you for all of your contributions!
+
+## Columnar Format Notes
+
+## Arrow Flight RPC notes
+
+In Java, applications can now enable zero-copy optimizations when writing data (ARROW-11066). This potentially breaks source compatibility, so it is not enabled by default.
+
+Arrow Flight is now packaged for C#/.NET.
+## C++ notes
+
+The Arrow C++ library now includes a [`vcpkg.json`](https://github.com/apache/arrow/blob/master/cpp/vcpkg.json) manifest file and a new CMake option `-DARROW_DEPENDENCY_SOURCE=VCPKG` to simplify installation of dependencies using the [vcpkg](https://github.com/microsoft/vcpkg) package manager. This provides an alternative means of installing C++ library dependencies on Linux, macOS, and Windows. See the [Building Arrow C++](https://arrow.apache.org/docs/developers/cpp/building.html) and [Developing on Windows](https://arrow.apache.org/docs/developers/cpp/windows.html) docs pages for details.
+## C# notes
+
+## Go notes
+
+## Java notes
+
+## JavaScript notes
+
+## Python notes
+
```

Comment:
- Writing parquet list types now has the option of enabling the canonical group naming according to the Parquet specification.

Reply:
```suggestion
- Writing parquet list types now has the option of enabling the canonical group naming according to the Parquet specification.

```